### PR TITLE
Fix the broken simnet configuration

### PR DESCRIPTION
**Pull request by MrPeteypoo**
Ensures that Breezmobile can connect to and benefit from the functionality of LSPD. Before these fixes the mobile application tries to connect to the wrong IP address so they can't receive incoming payments.

Thanks a lot to @roeierez for helping me out and fixing a lot of the issues I encountered

### CODE REVIEW

**Review by roeierez — CHANGES_REQUESTED** [review]
Thanks a lot @MrPeteypoo !
Now that the NODE_HOST is dynamic I think only thing needs to be added is the NODE_HOST to the simnet.yml lspd configuration with the old value.

**Comment by MrPeteypoo** [comment]
> Thanks a lot @MrPeteypoo ! Now that the NODE_HOST is dynamic I think only thing needs to be added is the NODE_HOST to the simnet.yml lspd configuration with the old value.

Good catch, I only looked at the dev-simnet stuff. Is there any differences I need to know with simnet.yml and start-network.sh as a whole?

**Comment by roeierez** [comment]
> Good catch, I only looked at the dev-simnet stuff. Is there any differences I need to know with simnet.yml and start-network.sh as a whole?

We use simnet.yml to run the integration tests and dev-simnet.yml to create a simnet environmentbe for an external mobile device.
They are pretty similar. I think the change I pointed is the only one you should address.

**Comment by MrPeteypoo** [comment]
> > Good catch, I only looked at the dev-simnet stuff. Is there any differences I need to know with simnet.yml and start-network.sh as a whole?
> 
> We use simnet.yml to run the integration tests and dev-simnet.yml to create a simnet environmentbe for an external mobile device. They are pretty similar. I think the change I pointed is the only one you should address.

If I understand the previous configuration correctly I believe I've updated simnet.yml with the correct value.

File: docker/simnet.yml [hunk 1]
```diff
@@ -149,7 +149,8 @@ services:
       context: ../
       dockerfile: docker/lspd/Dockerfile
     environment:
-      - NODE_PUBKEY
+      NODE_HOST: 10.5.0.3:9736
```
Comment (roeierez) [inline docker/simnet.yml:152]: I think it should be only 10.5.0.3 without the port as it uses the default port.
In different than dev-simnet.yml where there is port mapping to NODE_HOST, here we are using the internal network IP address so no port mapping is done between 9735 to 9736.

**Comment by MrPeteypoo** [comment]
Apologies for the confusing force pushes, vim decided it had to add a newline at the end of the file and so did gedit. I removed the port like you said.

**Review by roeierez — APPROVED** [review]
Thank you! LGTM

**Pull request merged**